sql: add Jaeger instructions to stmt bundle

tbg opened the pull request:
Once we really use this, it'll be nice for anyone to know what to do
when they're handed a `trace.txt` without having to waste their time
finding these instructions.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/47416)
<!-- Reviewable:end -->

tbg commented:
@andreimatei can you own backporting this along with the original change?

tbg committed:
sql: add Jaeger instructions to stmt bundle

Once we really use this, it'll be nice for anyone to know what to do
when they're handed a `trace.txt` without having to waste their time
finding these instructions.

Release note: None

andreimatei commented:
I've put these instructions in the trace file itself. Not good enough?

tbg commented:
I missed that, thanks for the heads up!

Posterity: https://github.com/cockroachdb/cockroach/blob/97bc2998b94fa5244b4e6fc3a3ac74fbdb12cea3/pkg/util/tracing/tracer_span.go#L516
